src/Voron/Data/Tables/AbstractSchemaIndexDefinition.cs

```diff
@@ -0,0 +1,38 @@
+using System;
```

Comment:
Note that the name of file is different `AbstractSchemaIndexDefinition.cs` than class defined here

Reply:
fixed